src/go/k8s/controllers/redpanda/redpanda_controller.go

```diff
@@ -64,11 +64,14 @@ type RedpandaReconciler struct {
 // flux additional resources
 // +kubebuilder:rbac:groups=source.toolkit.fluxcd.io,resources=buckets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=source.toolkit.fluxcd.io,resources=gitrepositories,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=apps,resources=replicasets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Why we are have CRUD operations for replicasets? Statefulset doesn't use them, right?